assets/fonts/circularBook/lineto-circular-book.svg

```diff
@@ -0,0 +1,346 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd" >

```

Comment:
I'd like to avoid having the fonts as part of the repo if possible.  Can we source them from adobe fonts instead? Or use similar, but not proprietary fonts?

Reply:
What if we try Bulma's default? And see how that looks?

Reply:
I think the Objektiv Mk2 and Mk3 families are close enough to Circular (https://fonts.adobe.com/fonts/objektiv). What do you think?

Reply:
Looks good to me!